README.md

```diff
@@ -1,15 +1,15 @@
-# Go framework for Cadence [![Build Status](https://badge.buildkite.com/e7241785444519bdfd1defc68839fd19a89c15adb3477c73f7.svg?theme=github&branch=master)](https://buildkite.com/uberopensource/cadence-go-client) [![Coverage Status](https://coveralls.io/repos/uber-go/cadence-client/badge.svg?branch=master&service=github)](https://coveralls.io/github/uber-go/cadence-client?branch=master) [![GoDoc](https://godoc.org/go.temporal.io/temporal?status.svg)](https://godoc.org/go.temporal.io/temporal)
+# Temporal GO SDK [![Build Status](https://badge.buildkite.com/ce6df3b1a8b375270261ae70fb2d2756af298fef3a0dac4d20.svg?theme=github&branch=master)](https://buildkite.com/temporal/temporal-go-client) [![GoDoc](https://godoc.org/go.temporal.io/temporal?status.svg)](https://godoc.org/go.temporal.io/temporal)
```

Comment:
I think we agreed to rename "go client" to "Go SDK" everywhere including repo name (should be `temporalio/temporal-go-sdk`). Let's do it on separate PR though.
nit: Should be "Go SDK", not "GO SDK".

Reply:
Agreed.